test/function/106.c

```diff
@@ -78,7 +78,7 @@ static void test(void *stack_pointer)
 
  die(mps_thread_reg(&thread, arena), "register thread");
  cdie(mps_root_create_thread(&root, arena, thread, stack_pointer), "thread root");
- die(mps_fmt_create_A(&format, arena, &fmtLO), "create format");
+ cdie(make_format_header(&format, arena), "create format");
```

Comment:
105.c and 107.c transformed mps_fmt_create_A into make_format, but this uses make_format_header, and fails to compile.  Format A doesn't have a header field, so this is *probably* a mistake.  Can you confirm @gareth-rees ?